### PR TITLE
ALL THE CHANGES AS PER REQUIRMENT

### PR DESCRIPTION
**Pull request by Sami-Ullah-Tufail**
- [x] Day 1
Milestone 0 - project setup (0.5h)

Set up the repository and tools.
Milestone 1 - content (1h)

Choose the topic for your website.
Choose images, text, icons, fonts (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
Milestone 2 - mobile first (5h)

Create the 2 pages for mobile.
Deploy the project.
Day 2
Milestone 3 - desktop version (5h)

Adapt the 2 pages to desktop.
Milestone 4 - interactions (1.5h)

Implement the user interactions: link, mobile menu.
Deploy the project.
Day 3
Milestone 5 - dynamic page (5h)

Implement the section "Featured speakers" with dynamic HTML.
Deploy the project and check to make sure everything works as planned.
Milestone 6 - documentation (1.5h)

Record a video describing your project.
Create a good README and pull request description.

### CODE REVIEW

**Comment by Sami-Ullah-Tufail** [comment]
My documentation presentation
https://www.loom.com/share/3be868caa58f4c3ba6c044d716e6cf9b

File: index.css [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+
+  footer {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 40px 40px;
+    color: #434342;
+    font-style: italic;
+    background: #fff;
+  }
```
Comment (mukhammadrizooff) [inline index.css]: - [ ] [REQUIRED] Good job so far! Please change the `background: #fff;` ❌ to correct one => `background: #d3d3d3;`✔️

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        <h2>Partners</h2>
+        <div class="h-line"></div>
+      </header>
+      <div class="container">
+        <img src="files/bafbd75750a1f1cadc82fbeb4d5eab8be4c8ef39_150.png" alt="BIMUN/SINUB 2022
+        Bonn, Germany">
+        <img src="files/053b6dae433ec8a2328c4f8584c03bcdd66b9988_150.png" alt="EPFLMUN 2023
+        Lausanne, Switzerland">
+      </div>
+    </section>
```
Comment (mukhammadrizooff) [inline index.html]: - [x] [REQUIRED] Good job so far! Please add `3` more `partners` as shown in the [original layout](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

image

File: README.md [hunk 3]
```diff
@@ -1 +1,229 @@
...
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank microverse and my coding partner
+https://github.com/misikira
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (mukhammadrizooff) [inline README.md]: - [ ] [REQUIRED] Good job so far! The `README` contains an "Acknowledgements" section that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 4]
```diff
@@ -1 +1,229 @@
...
+
+
+### Deployment
+
+The project is being deployed using the github pages
+
+**link**
+https://samiullah12345678.github.io/Portfolio-Website-design.github.io
```
Comment (mukhammadrizooff) [inline README.md]: - [x] [REQUIRED] Good job so far! Kindly your `Live Demo` link is not working, please replace the link with to correct one and check it before the submit your project 🙏🏼 

image

File: js/speakers.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
+const speakers = [{
```
Comment (mukhammadrizooff) [inline js/speakers.js:1]: - [x] [REQUIRED] Good job so far! Please remove `2` speakers and add the chess shape background to each speaker's avatar(picture) like [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)🙏🏼  

image

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Sami-Ullah-Tufail, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️
- Descriptive `PR` summary ✔️  

## Required Changes ♻️
- [x] [REQUIRED] Good job so far! I really appreciate what you did, but you forgot to talk about pages in `different screen sizes` and `highlight` a `piece of the code` that you find interesting or are proud of making in [your video presentation](https://www.loom.com/share/3be868caa58f4c3ba6c044d716e6cf9b).

image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/speakers.js (hunk 5 above)
Comment (Sami-Ullah-Tufail) [reply js/speakers.js:1]: I have made the changes in the speaker by removing them and for the design, the checkbox is not available in the design as the design is not publically available. and it should be acceptable as said  
Screenshot 2023-04-26 at 11 07 38 PM

File: index.css (hunk 1 above)
Comment (Sami-Ullah-Tufail) [reply index.css]: changed sir

File: index.html (hunk 2 above)
Comment (Sami-Ullah-Tufail) [reply index.html]: I did not have multiple sponsors but i copied the same three more t6imes to meet the requirements

File: README.md (hunk 3 above)
Comment (Sami-Ullah-Tufail) [reply README.md]: changes are made

**Comment by Sami-Ullah-Tufail** [comment]
New documentation link  
https://www.loom.com/share/3be868caa58f4c3ba6c044d716e6cf9b

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @Sami-Ullah-Tufail 👋, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Suggested changes
- Good job tendering a new presentation link, however the video presentation is the same with the previous one and you still didn't talk about a piece of code you are happy about neither did you show the website on mobile devices. Kindly do so to satisfy the project requirements.

_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Sami-Ullah-Tufail** [comment]
https://www.loom.com/share/1dd6d3bc82f94ad283f46dfca5f6704a soory i did that by mistake

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Sami-Ullah-Tufail , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**